code/controllers/subsystem/garbage.dm

```diff
@@ -37,6 +37,11 @@ var/datum/subsystem/garbage_collector/SSgarbage
 /datum/subsystem/garbage_collector/New()
 	NEW_SS_GLOBAL(SSgarbage)
 
+/datum/subsystem/garbage_collector/Shutdown()
+	//empty the queue
+	HandleToBeQueued(FALSE)
+	HandleQueue(null, FALSE)
```

Comment:
This would delay world shutdown/Reboot for almost a full minute.

at round end the queue can get really big and take forever to clear.

Destroy() is already called before something enters the queue or the tobequeued queue, so this isn't even needed.